internal/common/pointers.go

```diff
@@ -10,3 +10,13 @@ func FromPtr[T any](ref *T) (value T) {
 	}
 	return
 }
+
+// DerefOrDefault returns the dereferenced value of the given pointer or the
+// default value for the type if unset.
+func DerefOrDefault[T any](p *T) T {
+	var v T
+	if p != nil {
+		v = *p
+	}
+	return v
+}
```

Comment:
This method does literally what the one defined above does, named `FromPtr`:

https://go.dev/play/p/1Vu4HuSa_T9

Reply:
woops, right

Reply:
I think it was me who added it. On your defence, it lacks godoc completely it should have it perhaps.